commit d923fcf6361da3b8b25b13ce6c1e427e759f125a
Author: Jim Mattson <jmattson@google.com>
Date:   Thu Jun 1 12:43:37 2017 -0700

    KVM: nVMX: Don't update vmcs12->xss_exit_bitmap on nested VM-exit
    
    The XSS-exiting bitmap is a VMCS control field that does not change
    while the CPU is in non-root mode. Transferring the unchanged value
    from vmcs02 to vmcs12 is unnecessary.
    
    Signed-off-by: Jim Mattson <jmattson@google.com>
    Signed-off-by: Radim Krčmář <rkrcmar@redhat.com>

diff --git a/arch/x86/kvm/vmx.c b/arch/x86/kvm/vmx.c
index 39301297352a..fbc18085b599 100644
--- a/arch/x86/kvm/vmx.c
+++ b/arch/x86/kvm/vmx.c
@@ -10738,8 +10738,6 @@ static void sync_vmcs12(struct kvm_vcpu *vcpu, struct vmcs12 *vmcs12)
 	vmcs12->guest_sysenter_eip = vmcs_readl(GUEST_SYSENTER_EIP);
 	if (kvm_mpx_supported())
 		vmcs12->guest_bndcfgs = vmcs_read64(GUEST_BNDCFGS);
-	if (nested_cpu_has_xsaves(vmcs12))
-		vmcs12->xss_exit_bitmap = vmcs_read64(XSS_EXIT_BITMAP);
 }
 
 /*